### ISSUE TITLE
aria-relevant missing equivilant accessibilityRelevant

### ISSUE THREAD

**Issue by comp615** [necolas/react-native-web#1936]
**The problem**
Filing a bug so I don't lose this while working through aria-* issues. I noticed there was no equivalent conversion for aria-relevant, which changes how aria-live regions work.

**How to reproduce**
Using aria-* props to be removed in next version

**Expected behavior**
There should be a way to modify the behavior of aria-live

**Environment (include versions). Did this work in previous versions?**
Works in 0.15 and earlier

**Additional context**
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-relevant_attribute

**Comment by necolas** [comment]
AFAIK the current recommendation is not to use it and it may be deprecated in aria 1.4 https://github.com/w3c/aria/issues/712

**Comment by comp615** [comment]
Makes sense. Seems like `accessibilityAtomic` would do the same job then!
